Framework/PythonInterface/mantid/geometry/src/Exports/DetectorInfo.cpp

```diff
@@ -90,5 +90,9 @@ void export_DetectorInfo() {
 
       .def("rotation", rotation, (arg("self"), arg("index")),
            "Returns the absolute rotation of the detector where the detector "
-           "is identified by 'index'.");
+           "is identified by 'index'.")
+      .def("detectorIDs", &DetectorInfo::detectorIDs,
+           return_value_policy<
+               Mantid::PythonInterface::Policies::VectorToNumpy>(),
```

Comment:
Why is this not exposed as a read-only numpy array like, e.g., `MatrixWorkspace::readX`?

Reply:
Agreed will change this